### DISCUSSION TITLE
How to stop/start jobs during runtime?

### DISCUSSION THREAD

**Discussion by ClassyCircuit** [quartznet/quartznet#1271] [Q&A] [answered]
How can I stop a job and then start it again during runtime without restarting the application?

I want to build a switch that allows me to stop all jobs in a specific environment and then start them again on the same schedule as before.

**Comment by lahma** [comment, marked as answer]
`IScheduler.PauseJob` / `IScheduler.PauseTrigger` / `IScheduler.ResumeJob` / `IScheduler.ResumeTrigger` ?

**Reply by ClassyCircuit** [reply]
Thank you, this is exactly what I was looking for.
